automerge-backend/tests/apply_change.rs

```diff
@@ -694,7 +694,7 @@ fn test_handle_list_element_insertion_and_deletion_in_same_change() {
         seq: None,
         actor: None,
         max_op: 3,
-        deps: vec![change2.hash, change1.hash],
+        deps: vec![change1.hash, change2.hash],
```

Comment:
Good question. What has happened here is that the two changes are causally independent when they shouldn't be. This test should be written so that `change2` has `change1.hash` as a dependency and the final patch should just depend on `change2.hash`. I've rewritten the test to reflect this. The reason the order of the hashes changed was because the hash values changed (due to the new column types).